app/src/main/java/com/yeonae/chamelezone/view/course/CourseModifyActivity.kt

```diff
@@ -0,0 +1,262 @@
+        }
+
+        btn_place_add2.setOnClickListener {
+            PlaceCheckDialogFragment.newInstance(2).show(supportFragmentManager, "dialog")
+        }
+
+        btn_place_add3.setOnClickListener {
+            PlaceCheckDialogFragment.newInstance(3).show(supportFragmentManager, "dialog")
+        }
+
+        btn_close1.setOnClickListener {
+            firstPlaceNumber = NOT_SELECTED
+            tv_place_name1.text = ""
+            tv_place_keyword1.text = ""
+            tv_place_address1.text = ""
+            layout_course1.visibility = View.GONE
+            layout_place_add1.visibility = View.VISIBLE
+        }
+
+        btn_close2.setOnClickListener {
+            secondPlaceNumber = NOT_SELECTED
+            tv_place_name2.text = ""
+            tv_place_keyword2.text = ""
+            tv_place_address2.text = ""
+            layout_course2.visibility = View.GONE
+            layout_place_add2.visibility = View.VISIBLE
+        }
+
+        btn_close3.setOnClickListener {
+            thirdPlaceNumber = NOT_SELECTED
+            tv_place_name3.text = ""
+            tv_place_keyword3.text = ""
+            tv_place_address3.text = ""
+            layout_course3.visibility = View.GONE
+            layout_place_add3.visibility = View.VISIBLE
+        }
+
+        btn_register.setOnClickListener {
+            placeNumbers.clear()
+            if (firstPlaceNumber != NOT_SELECTED) {
+                placeNumbers.add(firstPlaceNumber)
+            }
+            if (secondPlaceNumber != NOT_SELECTED) {
+                placeNumbers.add(secondPlaceNumber)
+            }
+            if (thirdPlaceNumber != NOT_SELECTED) {
+                placeNumbers.add(thirdPlaceNumber)
+            }
+            when {
+                edt_course_title.text.isEmpty() -> shortToast(R.string.enter_course_title)
+                edt_course_content.text.isEmpty() -> shortToast(R.string.enter_course_content)
+                tv_place_name1.text.isEmpty() -> shortToast(R.string.select_two_places)
+                tv_place_name2.text.isEmpty() -> shortToast(R.string.select_two_places)
+                imageUri.isEmpty() -> shortToast(R.string.enter_course_image)
+            }
+            if (!isCreated) {
+                isCreated = true
+                presenter.registerCourse(
+                    memberNumber,
+                    placeNumbers,
+                    "${edt_course_title.text}",
+                    "${edt_course_content.text}",
+                    imageUri
+                )
+                Handler().postDelayed({
+                    isCreated = false
+                }, 1000)
+            }
+        }
+    }
+
+    private val permissionListener: PermissionListener = object : PermissionListener {
+        override fun onPermissionGranted() {
+            Toast.makeText(this@CourseModifyActivity, "권한이 허용되었습니다", Toast.LENGTH_SHORT).show()
```

Comment:
```suggestion
            Toast.makeText(this@CourseModifyActivity, HERE, Toast.LENGTH_SHORT).show()
```
해당 문구는 재사용될 문구로 보이니 리소스로 관리하는게 좋을 것 같아요.

Reply:
수정했습니다
55816fd